From f1111ce46894d3b51434e33a6f3461318dade5fd Mon Sep 17 00:00:00 2001
From: Rihards Skuja <rihards@skuja.eu>
Date: Thu, 29 Feb 2024 11:40:31 +0200
Subject: [PATCH] fix compile on gcc 13

Without the header compilation fails with a bunch of
"'uint8_t' does not name a type" errors.
---
 wifibroadcast/src/HelperSources/StringHelper.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/wifibroadcast/src/HelperSources/StringHelper.hpp b/wifibroadcast/src/HelperSources/StringHelper.hpp
index 056b12d..9d16c94 100644
--- a/wifibroadcast/src/HelperSources/StringHelper.hpp
+++ b/wifibroadcast/src/HelperSources/StringHelper.hpp
@@ -6,6 +6,7 @@
 #define OSDTESTER_STRINGHELPER_H
 
 #include <array>
+#include <cstdint>
 #include <sstream>
 #include <string>
 #include <vector>
-- 
2.44.0

